### PR TITLE
Replace cluster extra var with platform

### PR DESCRIPTION
**Pull request by rthallisey**
OpenShift removed the endpoint that identified it was openshift.  The Broker
doesn't seem to accurately identify which cluster it's running on so let's replace
the check for openshift.

```release-note
NONE
```

### CODE REVIEW

**Comment by rthallisey** [comment]
@lukas-bednar this PR fixed all the issue I had in the apb

File: roles/cdi/defaults/main.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ cdi_namespace: "{{ cdi_image_namespace | default('kube-system') }}"
 cdi_storageclass: "{{ cdi_kubevirt_storageclass | default('kubevirt') }}"
 apb_action: "provision"
 docker_prefix: "{{ cdi_repo_tag | default('kubevirt') }}"
-docker_tag: "{{ cdi_release_tag | default('v1.2.0') }}"
+cdi_release_tag: "v1.2.0"
```
Comment (lukas-bednar) [inline roles/cdi/defaults/main.yml]: I believe we should use cdi 1.3

Comment (nellyc) [reply roles/cdi/defaults/main.yml]: correct. @shiywang already has a PR for it

**Comment by rthallisey** [comment]
Seems like the cluster deployment failed? @gbenhaim 
```
Error from server (NotFound): users.user.openshift.io "test_admin" not found
```

**Comment by gbenhaim** [comment]
@rthallisey Adding:

```
   environment:
     http_proxy: ""
```

to the play will solve your problem. Without it `oc` tries to use the proxy.

**Comment by rthallisey** [comment]
ci test please

**Comment by gbenhaim** [comment]
@rthallisey 

```
{"command":"/usr/bin/oc get vmi vmi-with-sidecar-hook -o=jsonpath='{.status.phase}' -n kubevirt-test-default -n kubevirt-test-default","component":"","level":"error","msg":"command failed: cannot run command","output":"","pos":"utils.go:2004","reason":"exit status 1","stderr":"error: the server doesn't have a resource type \"vmi\"\n","timestamp":"2018-11-08T19:36:58.230841Z"}
```

How can it be?

**Comment by lukas-bednar** [comment]
@gbenhaim kubevirt wasn't deployed or kubevirt containers didn't rollout probably ?

**Comment by gbenhaim** [comment]
ci test please

**Comment by gbenhaim** [comment]
The node that run `virt-api` got `System OOM encountered`, might be related.
I'll post a patch to increase the memory of the nodes.

**Comment by gbenhaim** [comment]
@shiywang I think that the assertion in the second tests in `e2e_test` has a bug:

You can see the importer pod description:

https://jenkins.ovirt.org/job/kubevirt_kubevirt-ansible_standard-check-pr/1721/artifact/check-patch.openshift_3-10.el7.x86_64/obj-desc/after/e2e_test.go-59/kubevirt-test-default/pod/importer-golden-pvc1-cbntk/*view*/

The pod is running, but it's in a `CrashLoopBackOff`.

@rthallisey Since this is the only test the fails and if @shiywang approve that the test has a bug, I think that we can skip CI.

**Comment by rthallisey** [comment]
Merging this and we can sort out the CI issue later. Failure doesn't look related.

**Comment by rthallisey** [comment]
ci test please

**Pull request merged**